From c114cb18be04406d62a033ccce2fc38a71e2d144 Mon Sep 17 00:00:00 2001
From: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
Date: Tue, 17 Jul 2018 14:59:06 +0300
Subject: [PATCH] r8a7795-h3ulcb: Fix dclkin.2 clock

With "versaclock5 4" being used as dclkin.2 clock the HDMI1 interface
doesn't work properly (display which connected to HDMI1 claims that
there is no signal).

Signed-off-by: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
index 4f15df0..3b70272 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
@@ -94,7 +94,7 @@
 		 <&cpg CPG_MOD 727>,
 		 <&versaclock5 1>,
 		 <&versaclock5_out3>,
-		 <&versaclock5 4>,
+		 <&versaclock5_out3>,
 		 <&versaclock5 2>;
 	clock-names = "du.0", "du.1", "du.2", "du.3", "lvds.0",
 		      "dclkin.0", "dclkin.1", "dclkin.2", "dclkin.3";
-- 
2.7.4

